docs/self-hosted/kubernetes-migration-guide.md

```diff
@@ -148,15 +148,15 @@ it can take several minutes for this command to finish.
 ```
 export UI_POD=$(kubectl get pods --no-headers -o custom-columns=":metadata.name"|grep ui-)
 kubectl cp packagist_storage.tar.gz $UI_POD:/tmp/packagist_storage.tar.gz -c ui
-kubectl exec $UI_POD -c ui -- /bin/sh -c "/srv/manager/bin/console packagist:self-hosted:migrate-storage import /tmp/packagist_storage.tar.gz && rm /tmp/packagist_storage.tar.gz" 
+kubectl exec $UI_POD -c ui -- /bin/sh -c "/srv/manager/bin/console packagist:self-hosted:migrate-storage import /tmp/packagist_storage.tar.gz && rm /tmp/packagist_storage.tar" 
```

Comment:
The old command no longer works because the tarball gets gunzipped first. IF we do change this command, we should definitely also change minimum required version to 1.12.8 (see first change in this file). What's your thoughts on this?

Reply:
Sure, that sounds good to me!